src/Uno.UI/UI/Xaml/UIElement.Pointers.Android.cs

```diff
@@ -61,7 +65,7 @@ public bool OnTouch(View view, MotionEvent nativeEvent)
 
 		protected sealed override bool OnNativeMotionEvent(MotionEvent nativeEvent, View originalSource, bool isInView)
 		{
-			if (IsPointersSuspended)
+			if (ArePointersEnabled)
```

Comment:
Based on the name I would suggest:
```suggestion
			if (!ArePointersEnabled)
```